doc/script_commands.txt

```diff
@@ -6481,15 +6481,28 @@ Size is 0 = normal 1 = small 2 = big.
 *addtimer(<ticks>, "NPC::OnLabel"{, <account id>})
 *deltimer("NPC::OnLabel"{, <account id>})
 *addtimercount("NPC::OnLabel", <ticks>{, <account id>})
+*gettimer(<type>{, <account id>{, <event>}})
 
 These commands will create, destroy, and delay a countdown timer - 
-addtimer() to create, deltimer() to destroy and addtimercount() to delay
-it by the specified number of ticks. For all three cases, the event label 
-given is the identifier of that timer. The timer runs on the character 
-object that is attached to the script, and can have multiple instances.
-If <acccount id> is passed, this player will be used instead.
-When the label is run, it is run as if the player that the timer runs on 
-has clicked the NPC.
+addtimer() to create, deltimer() to destroy, addtimercount() to delay
+it by the specified number of ticks, and gettimer() to get misc info.
+For all three cases, the event label given is the identifier of that timer.
+The timer runs on the character object that is attached to the script,
+and can have multiple instances. If <acccount id> is passed, this player
+will be used instead. When the label is run, it is run as if the player
+that the timer runs on has clicked the NPC.
+
```

Comment:
although I have no idea who is the first to write the documentation for the timer script commands, combining them together look suck, each of them perform different tasks.

Maybe consider to split the documentations for these 4 script commands?